### ISSUE TITLE
Regression from earlier version, aria-colindex missing from table

### ISSUE THREAD

**Issue by pascal-pfeiffer** [h2oai/wave#2360]
#### Wave SDK Version, OS

Wave 1.3.4

#### Actual behavior

Somewhere between v1.0.2 and 1.3.4 the aria-colindex for tables was dropped. This information was especially helpful for automated UI testing, so it would be great to get it back. 

v1.0.2
![image](https://github.com/h2oai/wave/assets/1069138/df81b05e-f8fc-44e9-8341-adcfe4d2f30d)

v1.3.4:
![image](https://github.com/h2oai/wave/assets/1069138/d59942d5-a5bb-4851-bbb5-a9061b76b31d)

#### Expected behavior

aria-colindex being set for tables

**Comment by marek-mihok** [comment]
It was removed by FluentUI in version 8.52.1 according to its CHANGELOG:
```
{
      "date": "Tue, 01 Feb 2022 07:26:25 GMT",
      "tag": "@fluentui/react_v8.52.1",
      "version": "8.52.1",
      "comments": {
        "patch": [
          {
            "author": "sarah.higley@microsoft.com",
            "package": "@fluentui/react",
            "commit": "c4912723213bb1a15b4af2f6c7962bc32064bf4a",
            "comment": "remove aria-colindex from DetailsList columns by default, fix aria-label and aria-description on column headers"
          },
...
```

Looking to [docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-colindex) it does not seem to be necessary:
_If all of the columns are present in the DOM, including aria-colindex is not necessary as user agents can calculate the column index of each cell or gridcell. However, if any of the columns are left out of the DOM at any time, use aria-colindex to indicate the column of each cell or gridcell with respect to the full table._

The Wave regression lies in #2201 where we updated Fluent version from `8.49.0` to `8.58.0`

Looking to the [Fluent commit](https://github.com/microsoft/fluentui/commit/c4912723213bb1a15b4af2f6c7962bc32064bf4a), there is a new `data-item-key` property, but it only adresses column headers.

Is there any other way how you could adjust your tests? E.g. by querying the children of `ms-DetailsRow-fields` component?
